### PR TITLE
Feat/pulling

### PR DESCRIPTION
**Pull request by BiliouriV**
# PR Checklist 

### _1- PR Comments_
    *If needed, to be filled by developer*  

<br><hr/>
*WARNING : Do not edit the checklist below.*  
<hr/><br>

### _2- Developer section_

- [ ] [Documentation] is up to date with new feature
- [ ] [Tests] are *passed OK* (Non regression, new features & bug fixes)
- [ ] [Code Quality] Remind to check if :
    * Each function has a header (description, inputs, outputs) 
    * Code is commented (particularly in hard to understand areas)
    * There are no new warnings that can be corrected
    * Commits policy is respected (constitancy commits, clear commits comments)
<hr/><br>

### _3- QA section_

- [ ] [Review] tests for new features have been *reviewed*
- [ ] [Changelog] is up-to-date with expected tags  
    🆕 Feature: [Feature] Description...  
    🆕 Added: [Feature] Description...  
    🆕 Changed: [Feature] Description...  
    🛠️ Fix: [Feature] Description...

### CODE REVIEW

**Review by Salem-Luos — APPROVED** [review]

**Review by nicolas-rabault — APPROVED** [review]
This will be to improve with #273

File: engine/core/src/luos_engine.c [hunk 1]
```diff
@@ -1032,6 +1054,21 @@ uint16_t Luos_NbrAvailableMsg(void)
...
+/******************************************************************************
+ * @brief check if message of a specific topic is available
+ * @param service pointer
+ * @param topic_id
+ * @return true or false
+ ******************************************************************************/
+uint8_t Luos_FindTopicMsg(service_t *service, uint16_t topic_id)
+{
+    // search in msg alloc if we have a msg_task of a topic for this service
+    if (MsgAlloc_SearchLuosTaskTopic(service->ll_service, 0, topic_id) == SUCCEED)
```
Comment (JeromeGalan) [inline engine/core/src/luos_engine.c:1066]: Passing 0 to this function will write something at address 0. You should change that.

Comment (nicolas-rabault) [reply engine/core/src/luos_engine.c:1066]: @BiliouriV this is not a comment but a modification request. I think @JeromeGalan missclick on the PR state.
You should fix it to make this PR mergeable.

**Comment by nicolas-rabault** [comment]
This feature is not usefull for now.
This PR could be re-opened if needed

**Pull request closed without merging**